src/ontology/edits/oeo-import-edits.owl

```diff
@@ -98,7 +106,32 @@ Currently, it covers the `ro-extracted.owl`-file which is a subset of the Relati
 
     <!-- http://purl.obolibrary.org/obo/BFO_0000051 -->
 
-    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000051"/>
+    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000051">
+        <obo:IAO_0000233>issue https://github.com/OpenEnergyPlatform/ontology/issues/596
+Pull request https://github.com/OpenEnergyPlatform/ontology/pull/599
+Pull request https://github.com/OpenEnergyPlatform/ontology/pull/603
+
+fix duplicate implementation of `Inverse Of`
+issue https://github.com/OpenEnergyPlatform/ontology/issues/979
+pull request https://github.com/OpenEnergyPlatform/ontology/pull/1086</obo:IAO_0000233>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/IAO_0000136 -->
+
+    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000136">
+        <rdfs:domain rdf:resource="http://purl.obolibrary.org/obo/IAO_0000030"/>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/IAO_0000581 -->
+
+    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000581">
+        <rdfs:domain rdf:resource="http://purl.obolibrary.org/obo/IAO_0000582"/>
+        <obo:IAO_0000116>IAO declared &quot;time measurement datum&quot; as range, which is (not yet) imported by OEO. Therefore the range declaration is missing here.</obo:IAO_0000116>
```

Comment:
We don't need anymore this editor note as we import `time stamped measurement datum` now.

Reply:
`time stamped measurement datum` is not the same as `time measurement datum`!

Reply:
Sorry, you are right.